pkg/vsphere/toolbox/service.go

```diff
@@ -223,6 +223,10 @@ func (s *Service) SetOption(args []byte) ([]byte, error) {
 	case "broadcastIP": // TODO: const-ify
 		if val == "1" {
 			ip := s.PrimaryIP()
+			if ip == "" {
+				log.Printf("failed to find primary IP\n")
```

Comment:
That's is golang's log package so we don't have levels there. Not sure whether @dougm prefers it over logrus.